testplan/testing/multitest/result.py

```diff
@@ -1871,6 +1870,42 @@ def matplot(self, pyplot, width=2, height=2, description=None):
             description=description
         )
 
+    @bind_entry
+    def graph(self, graph_type, graph_data, description,
+              series_options, graph_options):
+        """
+        Displays a Graph in the report.
+
+         code-block:: python
+
+         result.graph('Line',
+                      {
+                          'graph 1':[{'x': 0, 'y': 8},{'x': 1, 'y': 5}]
+                      },
+                      description='Line Graph',
+                      individual_options={'graph 1':{"colour": "red"}},
+                      graph_options=None
+          )
+
+        :param graph_type: Type of graph user wants to create
+        :type graph_type: ``string``
+        :param graph_data: Data to plot on the graph, for each series.
+        :type graph_data: ``Dictionary[List[Dictionary]]``
```

Comment:
``dict`` instead of ``Dictionary`` please

Also technically the dict type requires two template types for the key and value like ``dict[KEY_TYPE, VAL_TYPE]``